bravado_asyncio/http_client.py

```diff
@@ -34,24 +34,19 @@
 log = logging.getLogger(__name__)
 
 
-#: module variable holding the current :class:`aiohttp.ClientSession`, so that we can share it between
-#: :class:`AsyncioClient` instances. Use :func:`get_client_session` to retrieve it and create one if none
-#: is present.
-client_session = None  # type: Optional[aiohttp.ClientSession]
-
-
 def get_client_session(loop: asyncio.AbstractEventLoop) -> aiohttp.ClientSession:
     """Get a shared ClientSession object that can be reused. If none exists yet it will
     create one using the passed-in loop.
 
     :param loop: an active (i.e. not closed) asyncio event loop
     :return: a ClientSession instance that can be used to do HTTP requests
     """
-    global client_session
-    if client_session:
+    try:
+        return loop._bravado_asyncio_client_session  # type: ignore
+    except AttributeError:
+        client_session = aiohttp.ClientSession(loop=loop)
+        loop._bravado_asyncio_client_session = client_session  # type: ignore
```

Comment:
I'm not sure setting an attribute directly on the loop object is "kosher" and MyPy doesn't like it.  Is there a better way to associate an object with a given event loop?